src/resources/drafts.ts

```diff
@@ -131,6 +132,15 @@ export class Drafts extends Resource {
         form,
```

Comment:
Same comment goes for this:
```suggestion
  const attachmentSize =
      requestBody.attachments?.reduce((total, attachment) => {
        return total + (attachment.size || 0);
      }, 0) || 0;
```